AEXS/aex-2.md

```diff
@@ -85,7 +87,9 @@ notifiy the SDK (and the app) that the wallet has been disconnected.
 
 The wallet can also let the SDK know of extra funtionality that it provides during the time of registration. These methods will vary from wallet to wallet and are out of the scope of this proposal.
 
-#### Workflow
+#### Workflow (Messages Exchange Example)
+
+We're using [JSON-RPC 2.0](https://www.jsonrpc.org/specification#examples) standard for messages exchange (already supported by aeternity's JS SDK), but for simplicity's sake, this documentation is not using the standard, so the actual messages exchange is slightly different.
```

Comment:
If you use the JSON-RPC standard then please also use it in this document. These documents should enable other implementers to produce an implementation without having to read much, if any, of the reference code

Reply:
updated.